{"kind":"symbol","hierarchy":{"paths":[["doc:\/\/GnustoEngine\/documentation\/GnustoEngine","doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ParseError"]]},"primaryContentSections":[{"kind":"declarations","declarations":[{"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"internalError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"platforms":["macOS"],"languages":["swift"]}]}],"variants":[{"paths":["\/documentation\/gnustoengine\/parseerror\/internalerror(_:)"],"traits":[{"interfaceLanguage":"swift"}]}],"metadata":{"externalID":"s:12GnustoEngine10ParseErrorO08internalD0yACSScACmF","roleHeading":"Case","modules":[{"name":"GnustoEngine"}],"symbolKind":"case","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"internalError","kind":"identifier"},{"text":"(","kind":"text"},{"preciseIdentifier":"s:SS","text":"String","kind":"typeIdentifier"},{"text":")","kind":"text"}],"role":"symbol","title":"ParseError.internalError(_:)"},"abstract":[{"type":"text","text":"An unexpected internal error occurred during parsing."}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ParseError\/internalError(_:)"},"schemaVersion":{"major":0,"patch":0,"minor":3},"sections":[],"references":{"doc://GnustoEngine/documentation/GnustoEngine/ParseError":{"role":"symbol","title":"ParseError","identifier":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ParseError","fragments":[{"kind":"keyword","text":"enum"},{"kind":"text","text":" "},{"kind":"identifier","text":"ParseError"}],"navigatorTitle":[{"kind":"identifier","text":"ParseError"}],"url":"\/documentation\/gnustoengine\/parseerror","kind":"symbol","abstract":[{"type":"text","text":"Enumerates errors that can occur during the command parsing phase."}],"type":"topic"},"doc://GnustoEngine/documentation/GnustoEngine":{"type":"topic","url":"\/documentation\/gnustoengine","abstract":[],"kind":"symbol","identifier":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine","title":"GnustoEngine","role":"collection"},"doc://GnustoEngine/documentation/GnustoEngine/ParseError/internalError(_:)":{"fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"internalError","kind":"identifier"},{"text":"(","kind":"text"},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":")","kind":"text"}],"title":"ParseError.internalError(_:)","abstract":[{"text":"An unexpected internal error occurred during parsing.","type":"text"}],"kind":"symbol","role":"symbol","url":"\/documentation\/gnustoengine\/parseerror\/internalerror(_:)","identifier":"doc:\/\/GnustoEngine\/documentation\/GnustoEngine\/ParseError\/internalError(_:)","type":"topic"}}}